unstoppable-ios-app/domains-manager-ios/Modules/Onboarding/OnboardingAddWallet/OnboardingAddWalletType.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  OnboardingAddWalletType.swift
+//  domains-manager-ios
+//
+//  Created by Oleg Kuplin on 23.04.2024.
+//
+
+import UIKit
+
+enum OnboardingAddWalletType: OnboardingStartOption {
+    case mpcWallet, selfCustody
+    
+    var icon: UIImage {
+        switch self {
+        case .mpcWallet:
+            return .shieldKeyhole
+        case .selfCustody:
+            return .walletExternalIcon
+        }
+    }
+    
+    var title: String {
+        switch self {
+        case .mpcWallet:
+            return "MPC Wallet"
+        case .selfCustody:
+            return String.Constants.selfCustody.localized()
+        }
+    }
+    
+    var subtitle: String? {
+        switch self {
+        case .mpcWallet:
+            return "Full cryptocurrency wallet for Web3 with enhanced backup."
```

Comment:
Extract all string constants in this file?

Reply:
This module is not finalised yet. Will be extracted when scope and copies finalised